icu4c/source/i18n/dtitvfmt.cpp

```diff
@@ -966,23 +966,26 @@ DateIntervalFormat::normalizeHourMetacharacters(const UnicodeString& skeleton) c
     
     UChar hourMetachar = u'\0';
     UChar dayPeriodChar = u'\0';
-    int32_t metacharStart = 0;
-    int32_t metacharCount = 0;
+    int32_t hourFieldStart = 0;
+    int32_t hourFieldLength = 0;
+    int32_t dayPeriodStart = 0;
+    int32_t dayPeriodLength = 0;
```

Comment:
How'd all this stuff get in here?  This is from my PR a week or so ago that fixed a bug in DateIntervalFormat.  I had to cherry-pick it from the maint-70 branch over to main.  I'm guessing this all got included in your PR by mistake.

Reply:
Gack, my fault with Git. I'll remove these extra files from the PR.

Reply:
I hate to say it, but my changes are still there.  I see my changes are showing as the first commit in this PR; maybe there's a way to rebase this on top of that so as to exclude that commit?

Reply:
Sorry to be slow fixing this PR. I'll try to fix this later today.

Reply:
No biggie.  It's happened to all of us...